app/cafe.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor.keys():
+            raise NotVaccinatedError
+
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError
+
+        if not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError
```

Comment:
The `NotWearingMaskError` should include a descriptive error message when raised. You can use `raise NotWearingMaskError("Visitor is not wearing a mask.")`.